package.json

```diff
@@ -32,7 +32,7 @@
     "test:p2p": "mocha -r ts-node/register test/p2p/*",
     "test:p2p:watch": "mocha -r ts-node/register test/p2p/*  --watch --watch-extensions ts",
     "test:crypto": "mocha -r ts-node/register test/crypto/*",
-    "test:sim": "(cd test/simulation && GO111MODULE=on go test -v)",
+    "test:sim": "(npm run compile && cd test/simulation && GO111MODULE=on go test -v)",
```

Comment:
Nice, saves me from having to manually recompile every time I run this locally!

Reply:
This came after I spend an hour trying to figure why the test are failing, after switching branches and not recompiling :\

Reply:
This came after I spend an hour trying to figure why the test are failing with a weird error, after switching branches and not recompiling :\